nats/testdata/certs/ca.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is it really needed ? we have this step for the other storages https://github.com/gofiber/storage/blob/1e017301cd303f880c93c658ad67d83b9c848294/.github/workflows/test-rueidis.yml#L27-L28

Reply:
Sorry, I just merged some recent changes. rueidis is not my work :)

Reply:
sorry was just dragged into another topic

yes, instead of checking in fixed certificates, we could do it like the other adapters and generate them on the fly

Reply:
@cr1cr1 can you adjust this ?

Reply:
Yes, working on it, but it is painful to test github actions locally.

Reply:
have you tried [act](https://github.com/nektos/act) 
with this i can use the workflow locally

f.e.
![image](https://github.com/gofiber/storage/assets/7063188/5a4295f7-e46e-47ce-84f9-848342915b0f)

Reply:
Yes, that's what I'm using but the generate files by the gen-test-certs.sh are not mounted in the nats-server docker using **act** because it seems they are generated in another container.